### ISSUE TITLE
Installation on domain subfolder 

### ISSUE THREAD

**Issue by vizo** [TypiCMS/Base#24]
What i need to configure to run it on http://domain.com/subfolder ?
I checked for Laravel and there you need just .htacces RewriteBase set to subfolder. 

I set up RewriteBase to /subfolder in .htacces, it works, but then i am redirected to root (http://domain.com/en).

**Comment by sdebacker** [comment]
I don't know exactly as I never had to do that.

**Comment by sdebacker** [comment]
I prefer to create a subdomain.

**Comment by sdebacker** [comment]
But It should be great to make it work in a subfolder.

**Comment by vizo** [comment]
I am trying to figure out, i will let you know ...

**Comment by vizo** [comment]
Ok. Looks like all you have to do is to add RewriteBase /subfolder and it works.

I think this is not related, but i still can't get working css frontend and backend bootstrap is missing ... If i look at generated css just fontawesome, modernizer and phpdebug classes are there and some others, no bootstrap.

**Comment by vizo** [comment]
Ok i figured out what is the problem... 
On frontend CSS is included in html with full absolute path (http:// ...), but in backend it's just relative from root (/build/css/admin-b529d7c098.css). Do you have any idea how to fix this?

**Comment by vizo** [comment]
Ok i figure it out, i will make pull request ...

**Comment by sdebacker** [comment]
You are right I closed the issue too early…

**Comment by sdebacker** [comment]
Is it working now ?

**Comment by vizo** [comment]
Nope :( I found out that i had to add in main views on frontend and backend in <head> tag
```html
<base href="{{ URL::to('/') }}/" />
```
and there is a lot of urls relative to root in views and js. All paths starts with '/'.
For all views is simple solution.
Just replace
```html
ng-include="'/
```
with
```html
ng-include="'
```
to get relative paths to base href.

I have just one problem more, angular $resource is calling /api/(module) instead of api/module. I changed it in javascript but it still calling e.g. /api/admin. I can't figure it out :(

I think after this all should work fine. With or without subfolder installation.
